docs/overview/what_is__-__main_areas/con-tracing-jaeger.md

```diff
@@ -0,0 +1,28 @@
+---
```

Comment:
short introduction to Jaeger, fine

Reply:
I guess we should provide one document with observability area in general and then explain the components 🤔